src/components/unit.tsx

```diff
@@ -34,21 +36,20 @@ const UnitStatus = (props: unitProps) => {
     <>
       {currentStatus !== STATUS_CODES.NOT_HOME && <>{status}</>}
       {currentStatus === STATUS_CODES.NOT_HOME && (
-        <Badge bg="secondary" text="light" className="me-1">
-          {status} {nhcount}
-        </Badge>
+        <span className="container-nothome">
+          <Image fluid src={envsvg} />
+          <Badge bg="none" className="badge-nothome" pill>
+            {nhcount}
+          </Badge>
+        </span>
       )}
       {note && <>🗒️ </>}
```

Comment:
<img width="68" alt="Screenshot 2022-11-21 at 10 33 10 PM" src="https://user-images.githubusercontent.com/40650158/203081299-5cee358b-7b26-43f9-b590-6154e139db35.png">

I think you may need to adjust your element spacing accordingly so that unit w nh , notes and type will line up nicely.